.github/workflows/static-analysis.yml

```diff
@@ -47,4 +47,4 @@ jobs:
 
     - name: Run cfn-lint
       # ingore "W3002 This code may only work with `package` cli command" warning
-      run: cfn-lint --ignore-checks W3002 --template *.yml **/*.yml
+      run: cfn-lint --ignore-checks W3002 E3006 --template *.yml **/*.yml
```

Comment:
Why ignore `E3006`?

Reply:
underscores are invaid in template names, we did it on purpose so that you cant deploy it without doing the replacement